tests/grpc/gov.go

```diff
@@ -0,0 +1,88 @@
+package grpc
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	proposal "github.com/cosmos/cosmos-sdk/x/params/types/proposal"
+	"github.com/ojo-network/ojo/client"
+)
+
+// SubmitAndPassProposal submits a proposal and votes yes on it
+func SubmitAndPassProposal(ojoClient *client.OjoClient, changes []proposal.ParamChange) error {
+	resp, err := ojoClient.TxClient.TxSubmitProposal(changes)
+	if err != nil {
+		return err
+	}
+
+	var proposalID string
+	for _, event := range resp.Logs[0].Events {
+		if event.Type == "submit_proposal" {
+			for _, attribute := range event.Attributes {
+				if attribute.Key == "proposal_id" {
+					proposalID = attribute.Value
+				}
+			}
+		}
+	}
+
+	if proposalID == "" {
+		return fmt.Errorf("failed to parse proposalID from %s", resp)
+	}
+
+	proposalIDInt, err := strconv.ParseUint(proposalID, 10, 64)
+	if err != nil {
+		return err
+	}
+
+	_, err = ojoClient.TxClient.TxVoteYes(proposalIDInt)
+	if err != nil {
+		return err
+	}
+
+	prop, err := ojoClient.QueryClient.QueryProposal(proposalIDInt)
+	if err != nil {
+		return err
+	}
+
+	now := time.Now()
+	sleepDuration := prop.VotingEndTime.Sub(now) + 3*time.Second
+	fmt.Printf("sleeping %s until end of voting period + 1 block\n", sleepDuration)
```

Comment:
Let's initialize a logger object in the grpc package since fmt.printf is used a couple times